docs/configuration.md

```diff
@@ -13,3 +13,11 @@ Certain features of the React application import variables from an [AppConfig](h
 ### Custom Fonts & Styles
 Fonts and css variables can be customized by modifying [fonts-custom.scss](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_fonts-custom.scss) and
 [variables-custom.scss](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_variables-custom.scss).
+
+### Examples
+
+#### 1) To add a custom logo you need to do the following:
+
+1. Add your logo to the folder in `amundsen_application/static/images/`
+2. Add the config for `logoPath`. For example Lyft uses the value  `"/static/images/lyft-logo.svg"`
```

Comment:
A more explicit direction about which file to edit  would also be helpful. Perhaps:
"Set the the `logoPath` key on the `AppConfigCustom` object in [custom-default.ts](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-custom.ts). For example Lyft uses the value  `/static/images/lyft-logo.svg`"

Reply:
Well it worked for the first test user who received the directions 🙂  - also being more explicit about location adds duplication/redundancy between code and doc. 

I think having the link to the repo location - now right above - is a good balance.